src/main/java/com/juick/android/SignUpActivity.kt

```
@@ -19,42 +19,47 @@ package com.juick.android
+                    onSignUp = { nick ->
+                        lifecycleScope.launch(Dispatchers.IO) {
+                            try {
+                                val user = App.instance.api.signup(nick, authCode)
+                                withContext(Dispatchers.Main) {
+                                    val successIntent = Intent()
+                                    successIntent.putExtra("nick", nick)
+                                    successIntent.putExtra("hash", user.hash)
+                                    setResult(RESULT_OK, successIntent)
+                                    finish()
+                                }
+                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)